baby-gru/src/utils/MoorhenMolecule.ts

```diff
@@ -192,6 +194,17 @@ export class MoorhenMolecule implements moorhen.Molecule {
         return Promise.reject(cootResponse.data.result.status)
     }
 
+    /**
+     * Turn on/off molecule symmetry
+     */
+    toggleBiomolecule() : void {
+        this.biomolOn = !this.biomolOn;
+        if(this.biomolOn)
+            this.symmetryOn = false;
```

Comment:
If we don't want cryst and bio symm at the same time, do we also need to have the same check in `toggleSymmetry` but the other way around?

Reply:
Done in 98e02208e3be2dfe1892e1d9bf9ac019e067b9d4.